charts/k8s-monitoring/templates/_crd-validation.tpl

```diff
@@ -2,13 +2,13 @@
 {{- $crdURL := printf "https://github.com/grafana/alloy-operator/releases/download/alloy-operator-%s/collectors.grafana.com_alloy.yaml" (index .Subcharts "alloy-operator").Chart.Version }}
 {{- if .Release.IsInstall }}
   {{- if not (.Capabilities.APIVersions.Has "collectors.grafana.com/v1alpha1/Alloy") }}
-    {{- if not (index .Values "alloy-operator").crds.deployAlloyCRD }}
+    {{- if not (dig "alloy-operator" "crds" "deployAlloyCRD" true .Values.AsMap) }}
       {{- $msg := list "" (printf "The %s Helm chart v3.0 requires the Alloy CRD to be deployed." .Chart.Name) }}
       {{- $msg = append $msg "Please set:" }}
       {{- $msg = append $msg "alloy-operator:" }}
       {{- $msg = append $msg "  crds:" }}
       {{- $msg = append $msg "    deployAlloyCRD: true" }}
-      {{- $msg = append $msg "" "Or install the Alloy CRD manually:" }}
+      {{- $msg = append $msg "Or install the Alloy CRD manually:" }}
```

Comment:
```suggestion
      {{- $msg = append $msg "" }}
      {{- $msg = append $msg "Or install the Alloy CRD manually:" }}
```

Reply:
My bad, append totally only takes one extra argument. For the sake of readability, can we add the newline back with this change?